index.js

```diff
@@ -218,7 +218,7 @@ exports.writeMainView = function (dir, cb) {
 exports.writeIcon = function (dir, cb) {
   var iconPath = path.join(__dirname, 'assets/icon.png')
```

Comment:
yay, thanks for testing! - I always thought it worked this way, but very happy to know it's actually the case :sparkles: